SPECS/httpd/httpd.spec

```diff
@@ -185,17 +190,16 @@ fi
 %{_bindir}/dbmmanage
 
 %changelog
-* Tue Aug 18 2020 Pawel Winogrodzki <pawelwi@microsoft.com> 2.4.46-1
-- Updated to 2.4.46 to resolve CVE-2020-11984.
-
-* Tue May 19 2020 Ruying Chen <v-ruyche@microsoft.com> 2.4.43-1
-- Updated to 2.4.43 to resolve the following CVEs
-- CVE-2019-10081, CVE-2019-10082, CVE-2019-10092, CVE-2019-10097
-- CVE-2019-10098, CVE-2020-1927, CVE-2020-1934
-
-* Sat May 09 00:20:57 PST 2020 Nick Samson <nisamson@microsoft.com> - 2.4.39-4
-- Added %%license line automatically
-
+*   Mon Sep 28 2020 Daniel McIlvaney <damcilva@microsoft.com> 2.4.46-2
+-   Mark CVE-1999-0236 CVE-1999-1412 as nopatch
+*   Tue Aug 18 2020 Pawel Winogrodzki <pawelwi@microsoft.com> 2.4.46-1
+-   Updated to 2.4.46 to resolve CVE-2020-11984.
+*   Tue May 19 2020 Ruying Chen <v-ruyche@microsoft.com> 2.4.43-1
+-   Updated to 2.4.43 to resolve the following CVEs
+-   CVE-2019-10081, CVE-2019-10082, CVE-2019-10092, CVE-2019-10097
+-   CVE-2019-10098, CVE-2020-1927, CVE-2020-1934
+*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> - 2.4.39-4
```

Comment:
nit: if you could remove the `-` before the version

Reply:
```suggestion
*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> 2.4.39-4
```